pkg/grizzly/workflow.go

```diff
@@ -34,34 +38,101 @@ func Get(config Config, dashboardUID string) error {
 
 // List outputs the keys of the grafanaDashboards object.
 func List(jsonnetFile string) error {
-	keys, err := dashboardKeys(jsonnetFile)
+	res, err := parse(jsonnetFile)
 	if err != nil {
 		return err
 	}
-	if len(keys) > 0 {
-		fmt.Println(yellow("Dashboards found in jsonnet:"))
+
+	f := "%s\t%s\n"
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 4, ' ', 0)
+
+	fmt.Fprintf(w, f, "KIND", "NAME")
+	for _, r := range res {
+		fmt.Fprintf(w, f, r.Kind(), r.UID())
 	}
-	for _, key := range keys {
-		fmt.Println(key)
+
+	return w.Flush()
+}
+
+type Group rulefmt.RuleGroup
```

Comment:
would it make sense to separate each resource type at least into its own file, if not package?

Reply:
Agreed. I will do a good cleanup once we have a working state. This will also include package refactoring, similar to how Tanka works